etc/config-template.toml

```diff
@@ -137,6 +137,15 @@
 [pd]
 ## PD endpoints.
 # endpoints = []
+# The interval at which to retry a PD connection initialization. Set to 0 to disable retrying 
+# connection initialization.
+# retry-interval = 300
```

Comment:
please keep the same style of the interval like "300ms" as other configurations do.